po/print-source-files.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
When running the script with `./print-source-files.sh` I get the following output:
```shell
data/com.mattjakeman.ExtensionManager.desktop.in.in
data/com.mattjakeman.ExtensionManager.metainfo.xml.in.in
./print-source-files.sh: 8: pushd: not found
find: ‘src’: Datei oder Verzeichnis nicht gefunden
find: ‘src’: Datei oder Verzeichnis nicht gefunden
./print-source-files.sh: 10: popd: not found
```

Running it with `bash print-source-files.sh` works without errors.

Reply:
Hm interesting, thanks for letting me know